demisto_sdk/commands/generate_docs/README.md

```diff
@@ -37,7 +37,12 @@ Skip generating of breaking changes section.
 * If no `output` given, the README.md file will be generated in the `input` file repository.
 * If no `additionalinfo` is provided for a commonly-used parameter (for example, `API Key`), a matching default value
   will be used, see the parameters and defaults in `default_additional_information.json`.
-
+* If an **incident mirroring** is implemented in an integration make sure that the following configuration params (if in use) are named as stated in order for the **incident mirroring** section to be generated:
```

Comment:
It will be better to check using the `name` and not the `display`, as it can vary.
It can be "Close Mirrored Jira **Ticket**" and not **incident** for exmaple.